### DISCUSSION TITLE
Turn off DummyHandler warning

### DISCUSSION THREAD

**Discussion by olischma** [Minestom/Minestom#1850] [Help]
Hey, I loaded a world with the anvilloader. In this world are many blocks that need a handler. 

Sample message:
`[ForkJoinPool.commonPool-worker-2] [14:21:24] (BlockManager.getHandlerOrDummy) - WARN - Block minecraft:bed does not have any corresponding handler, default to dummy.
You may want to register a handler for this namespace to prevent any data loss.
[ForkJoinPool.commonPool-worker-1] [14:21:24] (BlockManager.getHandlerOrDummy) - WARN - Block minecraft:mob_spawner does not have any corresponding handler, default to dummy.
You may want to register a handler for this namespace to prevent any data loss.
[ForkJoinPool.commonPool-worker-1] [14:21:25] (BlockManager.getHandlerOrDummy) - WARN - Block minecraft:banner does not have any corresponding handler, default to dummy.
`

I saw on Discord that you can somehow turn off the warnings, but have no idea about it. Another solution would certainly be to add a custom dummy handler for all the blocks. 
Can someone help me on this?
Thanks in advance

**Comment by mworzala** [comment]
Either register a handler on all the blocks like you said, or disable it at the logger. I am not sure how to do this with Tinylog, I dont use Tinylog.
